src/window.c

```diff
@@ -6,7 +6,7 @@
 
 u32 filler_03002F58;
 u32 filler_03002F5C;
-u8 gUnknown_03002F60;
+u8 gTransparentTileNumber;
```

Comment:
Should be static

Reply:
It cannot be defined as static, because it is used in the `window.txt` linker file. And it can't be removed from that file without causing a compiler error:
```
`gTransparentTileNumber' referenced in section `.text' of src/window.o: defined in discarded section `COMMON' of src/window.o
```